reference/configuration/framework.rst

```diff
@@ -2494,14 +2494,6 @@ enabled
 
 **type**: ``boolean`` **default**: ``true`` or ``false`` depending on your installation
 
-with_constructor_extractor
-..........................
-
-**type**: ``boolean`` **default**: ``false``
```

Comment:
Shouldn't we keep the option but change the default value here from `false` to `true` instead?